app/src/MyComponent.js

```diff
@@ -1,14 +1,18 @@
 import React from "react";
-import {
+import logo from "./logo.png";
+import { newContextComponents} from "@drizzle/react-components"
+
+const {
   AccountData,
   ContractData,
   ContractForm,
-} from "@drizzle/react-components";
+} = newContextComponents;
 
-import logo from "./logo.png";
 
-export default ({ accounts }) => (
-  <div className="App">
+export default (props) => {
+  const { drizzle, drizzleState } = props;
```

Comment:
Could move this up to the function args area, I wouldn't say this is mandatory though.

```
export default ({ drizzle, drizzleState }) => (
  <div className="App>
  ...
)
```